### PR TITLE
Add library eModal@1.2.65 w/ npm auto-update

### PR DESCRIPTION
**Pull request by LuisRBarreras**
**Add library eModal with version 1.2.65**
- files eModal.js and eModal.min.js
- npm auto-update. 

Pull request for issue: #10119 

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required):
 * Official website (optional, not the repository):
 * NPM package url (optional):
 * License and its reference:
 * GitHub / Bitbucket popularity (required):
   - Count of watchers:
   - Count of stars:
   - Count of forks:
 * NPM download stats (optional):
   - Downloads in the last day:
   - Downloads in the last week:
   - Downloads in the last month:

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [ ] This lib was not found on cdnjs repo
 * [ ] No already exist / duplicated issue and PR
 * [ ] The lib has notable popularity
   * [ ] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [ ] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/e-modal/package.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    "type": "git",
+    "url": "https://github.com/saribe/eModal"
+  },
+  "license": "MIT",
+  "npmName": "emodal",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "*.js"
```
Comment (hare1039) [inline ajax/libs/e-modal/package.json]: We better use `**/*` for dist folders, because it is more flexible.

File: ajax/libs/e-modal/package.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  "title": "eModal is a Easy way to manage modal dialogs using bootstrap.",
+  "description": "Easy Modal for bootstrap, is a simple way to create modal dialogs using javaScript.",
+  "author": "Samuel Ribeiro Pinto <samuel@maispc.com>",
+  "version": "1.2.65",
+  "keywords": [
+    "bootstrap",
+    "modal",
+    "dialog"
+  ],
+  "homepage": "http://saribe.github.io/eModal/",
```
Comment (hare1039) [inline ajax/libs/e-modal/package.json]: I suggest to use https

File: ajax/libs/e-modal/package.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
+{
+  "name": "eModal",
+  "filename": "eModal.min.js",
+  "title": "eModal is a Easy way to manage modal dialogs using bootstrap.",
```
Comment (hare1039) [inline ajax/libs/e-modal/package.json]: We don't need `"title"` field, can you remove this line?

File: ajax/libs/e-modal/package.json [hunk 4]
```diff
@@ -0,0 +1,28 @@
+{
+  "name": "eModal",
```
Comment (hare1039) [inline ajax/libs/e-modal/package.json]: I think we better follow original name: `emodal` https://github.com/saribe/eModal/blob/master/package.json#L35

Comment (hare1039) [inline ajax/libs/e-modal/package.json]: Also apply to the folders too

Comment (extend1994) [inline ajax/libs/e-modal/package.json]: I think we can just follow the library request. So `eModal` is okay.

Comment (extend1994) [inline ajax/libs/e-modal/package.json]: name in package.json should be parent folder name. In this case, my suggestion is that you can modify the folder name from e-modal to eModal.

**Pull request closed without merging**